python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -557,30 +557,48 @@ def test_elasticnet(
         ("elasticnet", 2.0, 0.2),
     ],
 )
-@pytest.mark.parametrize("datatype", [np.float32])
+@pytest.mark.parametrize("datatype", [np.float32, np.float64])
 @pytest.mark.parametrize("delayed", [True])
 @pytest.mark.parametrize("n_classes", [2, 8])
 def test_sparse_from_dense(
     fit_intercept, regularization, datatype, delayed, n_classes, client
 ):
-    penalty = regularization[0]
-    C = regularization[1]
-    l1_ratio = regularization[2]
+    penalty, C, l1_ratio = regularization
 
-    test_lbfgs(
-        nrows=1e5,
-        ncols=20,
-        n_parts=2,
-        fit_intercept=fit_intercept,
-        datatype=datatype,
-        delayed=delayed,
-        client=client,
-        penalty=penalty,
-        n_classes=n_classes,
-        C=C,
-        l1_ratio=l1_ratio,
-        convert_to_sparse=True,
-    )
+    if datatype == np.float32:
+        test_lbfgs(
```

Comment:
We can avoid needing to duplicate the whole function call with `functools.partial`:
```
from functools import partial

...

run_test = partial(test_lbfgs, ....)

if datatype == np.float32:
    run_test()
else:
    with pytest.raises(...):
        run_test()
```